Bots/JavaScript/Consumers/CodeFirst/SimpleHostBot/.env

```diff
@@ -2,6 +2,20 @@ MicrosoftAppId=
 MicrosoftAppPassword=
 SkillHostEndpoint=http://localhost:3978/api/skills/
 
-SkillId=EchoSkillBot
-SkillAppId=TODO: Add here the App ID for the skill
-SkillEndpoint=http://localhost:39783/api/messages
+skill_EchoSkillBotDotnet_appId=TODO: Add here the App ID for the skill
```

Comment:
Do not add a TODO, leave blank so the bot can be tested locally without having to edit the env file (remove this in the entire env file)